connect/api/src/main/java/org/apache/kafka/connect/storage/StringConverter.java

```diff
@@ -104,6 +104,7 @@ public SchemaAndValue toConnectHeader(String topic, String headerKey, byte[] val
 
     @Override
     public void close() {
-        // do nothing
+        this.serializer.close();
+        this.deserializer.close();
```

Comment:
Good catch here. Should we do the same in the [NumberConverter](https://github.com/apache/kafka/blob/30216ea1c58761e62f51af40033f24e3ae1c5c2a/connect/runtime/src/main/java/org/apache/kafka/connect/converters/NumberConverter.java#L123-L125) class?

Reply:
Also, even though these are no-ops right now, if we want to be meticulous about the cleanup logic here, should we wrap both of these in `Utils::closeQuietly`, so that if one fails, we still get a chance to close the other?